source/2019-10-18-the-ember-times-issue-120.md

```diff
@@ -54,13 +54,10 @@ Be sure to check out the future episodes as they are released!
 
 ---
 
-## [Section Title in Title Case 🐹](#section-url)
-
-<change section title emoji>
-<consider adding some bold to your paragraph>
-
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+## [Ember-engines: the what, what not and why? – Part 1 📝](https://medium.com/developer-paradise/ember-engines-the-what-what-not-and-when-part-1-49187c949db5)
+  
+If you have ever heard something about **ember-engines** but you need to refresh your knowledge or they are completely new to you [@abhilashlr](https://github.com/abhilashlr) wrote a [blogpost](https://medium.com/developer-paradise/ember-engines-the-what-what-not-and-when-part-1-49187c949db5) about them. This first [blogpost](https://medium.com/developer-paradise/ember-engines-the-what-what-not-and-when-part-1-49187c949db5) nicely establishes what an ember-engine is, and emphasises what it is not. 
+The [blogpost](https://medium.com/developer-paradise/ember-engines-the-what-what-not-and-when-part-1-49187c949db5) is part one, with part two on how  [@abhilashlr ](https://github.com/abhilashlr) used ember engines for one of his features in draft. So keep posted to learn more about how to use ember-engines, especially in large scale Ember apps.
```

Comment:
```suggestion
Stay tuned for part two on how [@abhilashlr](https://github.com/abhilashlr) used ember-engines in his app.
```

took a first pass at this, but need to think about it more, open to discussion! also if you are already gone emberfest-ing, will finish it off. thank you for the post this week and have a great time at emberfest!!